src/fairseq2/recipes/lm/_online_finetune/_recipe.py

```diff
@@ -297,7 +297,8 @@ def load_online_finetuner(
 
     unit = unit_handler.create(model, gangs, config, vllm_actors)
     try:
-        unit.maybe_sync_models(force_sync_vllm=True)
+        if unit._sync_vllm_model_every_n_steps >= 0:
```

Comment:
not relevant to rollout lengths, but this should only be done if we're doing online/semi-online training.